docs/src/operations/requirements.md

```diff
@@ -18,50 +18,18 @@ transaction for each block the validator agrees with, which can cost up to
-  - Snapshots: 250GB or larger. High TBW suggested
-  - OS: (Optional) 500GB, or larger. SATA OK
-  - The OS may be installed on the ledger disk, though testing has shown better
-    performance with the ledger on its own disk
-  - Accounts and ledger _can_ be stored on the same disk, however due to high
-    IOPS, this is not recommended
-  - The Samsung 970 and 980 Pro series SSDs are popular with the validator community
-- GPUs
-  - Not necessary at this time
-  - Operators in the validator community do not use GPUs currently
-
-### RPC Node Recommendations
-
-The [hardware recommendations](#hardware-recommendations) above should be considered
-bare minimums if the validator is intended to be employed as an RPC node. To provide
-full functionality and improved reliability, the following adjustments should be
-made.
-
-- CPU
-  - 16 cores / 32 threads, or more
-- RAM
-  - **1.17** 512 GB or more if an `account-index` is used, 1TB+ for all three [account indexes](https://docs.solanalabs.com/operations/setup-an-rpc-node#account-indexing)
-  - **1.18 or newer** 512 GB or more for all three indexes
-- Disk
-  - Consider a larger ledger disk if longer transaction history is required
-  - Accounts and ledger should not be stored on the same disk
+### Hardware Recommendations
+
+| Component | Validator Requirements | Additional RPC Node Requirements |
+|-----------|------------------------|----------------------------------|
+| **CPU**   | 2.8GHz base clock speed, or faster<br>SHA extensions instruction support<br>AMD Gen 3 or newer<br>Intel Ice Lake or newer<br>Higher clock speed is preferable over more cores<br>AVX2 instruction support (to use official release binaries, self-compile otherwise)<br>Support for AVX512f is helpful ||
+| **CPU**   | 12 cores / 24 threads, or more| 16 cores / 32 threads, or more |
+| **RAM**   | Error Correction Code (ECC) memory is suggested<br>Motherboard with 512GB capacity suggested ||
+| **RAM**   | 256GB or more| **1.17**:  512 GB or more if **one** `account-index` is used, 1TB+ **for all three** [account indexes](https://docs.solanalabs.com/operations/setup-an-rpc-node#account-indexing)<br>**1.18 or newer**:  512 GB or more for **all three indexes** |
+| **Disk**  | PCIe Gen3 x4 NVME SSD, or better ||
+| **Disk**  | **Accounts**: 500GB, or larger. High TBW (Total Bytes Written)<br>**Ledger**: 1TB or larger. High TBW suggested<br>**Snapshots**: 250GB or larger. High TBW suggested<br>**OS**: (Optional) 500GB, or larger. SATA OK<br><br>The OS may be installed on the ledger disk, though testing has shown better performance with the ledger on its own disk<br><br>Accounts and ledger *can* be stored on the same disk, however due to high IOPS, this is not recommended<br><br>The Samsung 970 and 980 Pro series SSDs are popular with the validator community | Consider a larger ledger disk if longer transaction history is required<br><br>Accounts and ledger **should not** be stored on the same disk |
```

Comment:
Combine these into one disk row ?

With that, I think we could adjust the comment:
"PCI Gen3 x4 NVME SSD, or better. One for each of:"

This would be a nice lead into the breakout of the 4 drives

Reply:
Per our DM's - we'll leave this as two rows for the visual difference, but drop "Disk" from the second row